client/src/templates/Challenges/classic/editor.tsx

```diff
@@ -273,9 +275,9 @@ const Editor = (props: EditorProps): JSX.Element => {
   // reference to *those* props. If we want it to use the latest props, we can
   // use a ref, since it will be updated on every render.
   const testRef = useRef<Test[]>([]);
-  testRef.current = props.tests;
+  testRef.current = propsRef.current.tests;
```

Comment:
```suggestion
  testRef.current = props.tests;
```
It's fine to use props directly here.